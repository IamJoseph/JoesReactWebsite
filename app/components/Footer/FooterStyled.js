import styled from 'styled-components';

export const Container = styled.footer`
  align-items: center;
  bottom: 0;
  display: flex;
  height: 15px;
  justify-content: center;
  pointer-events: none;
  position: fixed;
  width: 100%;
  @media(min-width: 768px){
    height: 35px;
  }
`;

export const InnerContainer = styled.div`
  align-items: center;
  display: flex;
  height: 35px;
  justify-content: flex-end;
  margin-left: 25px;
  margin-right: 25px;
  max-width: 1460px;
  pointer-events: none;
  width: 100%;
`;

export const Email = styled.div`
  background-color: rgba(238, 236, 226, .5);
  color: #686664;
  font-size: .7em;
  margin-right: 2em;
`;
