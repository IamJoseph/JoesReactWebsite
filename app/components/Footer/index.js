/**
*
* Footer
*
*/

import React from 'react';
import { Container, InnerContainer, Email } from './FooterStyled';

export default function Footer() {
  return (
    <Container>
      <InnerContainer>
        <Email>email: iamjoe@joeireland.com</Email>
      </InnerContainer>
    </Container>
  );
}
