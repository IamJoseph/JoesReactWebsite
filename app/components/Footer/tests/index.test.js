import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from '../index';

describe('<Footer />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
