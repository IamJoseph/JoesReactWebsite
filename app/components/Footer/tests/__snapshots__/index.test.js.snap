// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders correctly 1`] = `
<styled.footer>
  <styled.div>
    <styled.div>
      email: iamjoe@joeireland.com
    </styled.div>
  </styled.div>
</styled.footer>
`;
